<?xml version="1.0"?>

<launch>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find stair_complete)/config/offline_opt_visual.rviz" />

    <param name="tsdf_save_path" value="$(find stair_complete)/saved_layer/big_truncation.vxblx" />

    <node name="tf_world_set" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 1 world odom 10" />

    <node pkg="stair_complete" type="baseline_node" name="baseline_node" output="screen" launch-prefix="gnome-terminal -e"/>

    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" />

</launch>
